Tests/FunctionalTests/fss-simplified-search-results.spec.ts

```diff
@@ -93,77 +93,81 @@ describe('Test Search Result Scenario On Simplified Search Page', () => {
      expect(fileDownloadStatus).toContain("check");    
   })
 
-  it('Verify search results specific batch attributes on filter panel', async () => {
+  it('Verify search results specific batch attributes Not displayed on filter panel', async () => {
```

Comment:
Please rename this test to `Verify batch attributes with single value are not displayed on filter panel`
```suggestion
  it('Verify batch attributes with single value are not displayed on filter panel', async () => {
```

Reply:
Resolved. Thanks